packages/flat-lists/package.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "@react-native/flat-lists",
+    "version": "1.0.0",
+    "description": "Flat lists for React Native.",
+    "repository": {
+      "type": "git",
+      "url": "git@github.com:facebook/react-native.git",
+      "directory": "packages/flat-lists"
+    },
+    "license": "MIT",
+    "peerDependencies": {
+      "react": "18.2.0",
+      "react-native": "*",
+      "@react-native/virtualized-lists": "1.0.0",
+      "react-test-renderer": "18.2.0"
```

```suggestion
      "react-native": "*",
      "@react-native/virtualized-lists": "1.0.0",
```

Or do you have a specific reason to have a peerDependency on react?

---

frankly, not sure.. interacting 1st time with these libraries :)
`react` is used in `FlatList`s components as well as related tests, if it was separate package (repo) itself I would've kept it, not sure how deps scoping and resolving will work in monorepo (?)

---

I believe you should be fine by setting the peerDependency on RN only. RN is providing react for you and you'll be good. Specifying a React version here adds unnecessary complexity for us.

---

Thanks for letting me know :)